src/core_ct/slice.py

```diff
@@ -32,15 +32,15 @@ def __init__(self, data: np.ndarray, pixel_dimensions: tuple[float, float]):
         self.data: np.ndarray = data
         self.pixel_dimensions: tuple[float, float] = pixel_dimensions
 
-    def trim(self, axis: int, loc_start: int, loc_end: int | None = None) -> Slice:
+    def trim(self, axis: int, left_index: int, right_index: int | None = None) -> Slice:
         """
         Create new slice by trimming off a specified amount on the requested axis.
```

Comment:
Maybe this should say something more like "Create new slice containing trimmed data" or something. This is a nitpick, I might just be a little iffy on "create new by removing data"